packages/neuron-ui/.prettierrc.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  tabWidth: 2,
+  useTabs: false,
```

Comment:
`useTabs: true` means using `\t` instead of spaces. By default, `useTabs` is false, which means using spaces.
We have been using spaces before and should not make changes, otherwise all files will conflict.
I just explicitly declared it for safety instead of using the default value. Perhaps this is unnecessary.